src/main/java/com/questhelper/statemanagement/BarbarianTrainingStateTracker.java

```diff
@@ -0,0 +1,312 @@
+					new MesBoxRequirement("You feel you have learned more of barbarian ways. Otto might wish to talk to you more.")
+				),
+				new WidgetTextRequirement(119, 3, true,
+					"I've managed to catch a <col=800000>fish with the new rod<col=000080>! I should let")
+			)
+		);
+
+		caughtFishWithoutHarpoon = new RuneliteRequirement(configManager, "barbariantrainingharpoonedfish",
+			new Conditions(true, LogicType.OR,
+				new MultiChatMessageRequirement(
+					new ChatMessageRequirement("You catch a tuna.", "You catch a swordfish.", "You catch a shark.", "You catch a shark!"),
+					new MesBoxRequirement("You feel you have learned more of barbarian ways. Otto might wish to talk to you more.")
+				),
+				new WidgetTextRequirement(119, 3, true,
+					"I've managed to <col=800000>fish with my hands<col=000080>! I should let <col=800000>Otto <col=000080>know")
+			)
+		);
+		madePotion = new RuneliteRequirement(configManager, "barbariantrainingmadepotion",
+			new Conditions(true, LogicType.OR,
+				new MultiChatMessageRequirement(
+					new ChatMessageRequirement("You combine your potion with the fish eggs."),
+					new MesBoxRequirement("You feel you have learned more of barbarian ways. Otto might wish to talk to you more.")
+				),
+				new WidgetTextRequirement(119, 3, true,
+					"I've managed to make a <col=800000>new type of potion<col=000080>! I should let")
+			)
+		);
+		madeSpear = new RuneliteRequirement(configManager, "barbariantrainingmadespear",
+			new Conditions(true, LogicType.OR,
+				new MultiChatMessageRequirement(
+					new ChatMessageRequirement("You make a "),
+					new ChatMessageRequirement(" spear."),
+					new MesBoxRequirement("You feel you have learned more of barbarian ways. Otto might wish to talk to you more.")
+				),
+				new WidgetTextRequirement(119, 3, true,
+					"I've managed to <col=800000>smith a spear<col=000080>!")
+			)
+		);
+
+		madeHasta = new RuneliteRequirement(configManager, "barbariantrainingmadehasta",
+			new Conditions(true, LogicType.OR,
+				new MultiChatMessageRequirement(
+					new ChatMessageRequirement("You make a "),
+					new ChatMessageRequirement(" hasta."),
+					new MesBoxRequirement("You feel you have learned more of barbarian ways. Otto might wish to talk to you more.")
+				),
+				new WidgetTextRequirement(119, 3, true,
+					"I've managed to <col=800000>smith a hasta<col=000080>!")
+			)
+		);
+
+		reqs = new RequirementValidator(client, eventBus, (or(
+			taskedWithFishing, taskedWithHarpooning, taskedWithFarming, taskedWithBowFiremaking, taskedWithPyre, taskedWithPotSmashing,
+			taskedWithSpears, taskedWithHastae, taskedWithHerblore, plantedSeed, smashedPot, litFireWithBow, sacrificedRemains, caughtBarbarianFish,
+			caughtFishWithoutHarpoon, madePotion, madeSpear, madeHasta, finishedFishing, finishedHarpoon, finishedSeedPlanting, finishedPotSmashing,
+			finishedFiremaking, finishedPyre, finishedSpear, finishedHasta, finishedHerblore
+		)));
```

Comment:
It looks weird to me that the requirements are passed in under an `or(` here. Would it be the same functionally if `RequirementValidator` instead accepted a `List<Requirement>` and we ran `List.of(...)` of all the requirements here?